quiz.js

```diff
@@ -37,9 +52,17 @@ db.dropDatabase();
 
 // your code begins here
 
+db.houses.update(
+   { house: stark },
+   { $push: { respective houses: 'Ned Stark', 'Arya Stark','Sansa Stark' }}
 
+db.houses.update(
+   { house: Targaryen },
+   { $push: { respective houses: 'Viserys Targaryen', 'Daenerys Targaryen'}}
 
-
+db.houses.update(
+   { house: Arryn },
+   { $push: { respective houses: 'Jon Arryn'}}
 
 // your code ends here
```

Comment:
Unfortunately, this is not the right syntax for doing an $push. To $push a single value, you would need to write something like this
```
db.houses.update({ house : 'Arryn' }, { $push : { members :  'Jon Arryn'
// or  { name : 'Jon Arryn' } if you wanted to push an object instead of just a string.
} });
```
To $push multiple values at once, you can use the $each operator. 
```
db.houses.update({ house : 'Stark' }, { $push : { members : 
  { $each : [
    { name : 'Ned Stark' }
    { name : 'Arya Stark' }
    { name : 'Sansa Stark' }
  ] }
} });
```